README.md

```diff
@@ -391,6 +405,13 @@ module.exports = result;
 
 ---
 
+## Validation and Sanitization are different
+
+Validation and Sanitization of a request are two different process. Do them seprately. Although all the sanitizer methods canbe used during validation but those sanitizations will be effective for the validation purpose only. That sanitization will not persist. 
+For persistent sanitization of request it's must to use 'express-validator/filter' methods.
```

Comment:
I saw many cases of people getting astonished with such behaviour. It definitely needs to be changed (as a breaking change), so I'm not keen on adding it to the docs now that the next release is due soon.

Reply:
Cool.